parsing/gameData.json

```diff
@@ -501,11 +502,11 @@
                 "energyGeneratedInMJ": 750
             },
             "LiquidOil": {
-                "name": "Liquid Oil",
+                "name": "Crude Oil",
                 "stackSize": 0,
                 "isFluid": true,
                 "isFicsmas": false,
-                "energyGeneratedInMJ": 0
+                "energyGeneratedInMJ": 320
```

Comment:
We sure this actually generates any energy? Can it be burned directly?

Reply:
You are correct - today it can not be burned directly - however, it has this value in the game data. Given this, it is possible it could happen in the future. The goal here was to match the game data files - who knows what is coming in 1.1.

Reply:
wiki entry for a visual confirmation: https://satisfactory.wiki.gg/wiki/Crude_Oil

Reply:
reference in original data: 

![image](https://github.com/user-attachments/assets/2e5ffedf-5116-41c6-a3a7-f94b50500316)